cypress/e2e/registration.cy.js

```diff
@@ -0,0 +1,56 @@
+/// <reference types='cypress' />
+  const username = 'qa_test22';
+  const password = '123Qwert!';
+
+describe('cypress forth HW', () => {
+
+
```

Comment:
no need to add 2 empty lines here. make your code readable, but do not overdo it 🙂